tools/ios/MASTG-TOOL-0046.md

```diff
@@ -2,6 +2,8 @@
 title: Cycript
```

Comment:
adding a reason would be nice.

Something like..?

> Cycript is no longer actively maintained and fails on modern iOS versions. It last saw meaningful updates between 2009 and 2013. Key components like cynject broke on iOS 12 around 2019 due to changes in Cydia Substrate and have not been fixed. Frida offers broader compatibility, active support, and more powerful dynamic instrumentation capabilities. The OWASP MASTG recommends using Frida for reliable and effective iOS security testing.

Reply:
Fixed.